currency/storage.go

```diff
@@ -369,37 +369,42 @@ func (s *Storage) WriteCurrencyDataToFile(path string, mainUpdate bool) error {
 }
 
 // LoadFileCurrencyData loads currencies into the currency codes
-func (s *Storage) LoadFileCurrencyData(f File) error {
+func (s *Storage) LoadFileCurrencyData(f *File) error {
 	for _, contract := range f.Contracts {
```

Comment:
Might be able to dereference the pointer and get the mem address of the item so we don't double copy like so, might not work correctly?
```go
for i := range f.Contracts {
		err := s.currencyCodes.LoadItem(&(*f).Contracts[i])
		if err != nil {
			return err
		}
	}
```